tests/integrationtest/t/globalindex/analyze.test

```diff
@@ -0,0 +1,70 @@
+set tidb_enable_global_index=true;
```

Comment:
Could you please provide a case to cover the analysis of multiple global indexes here?